codec-http2/src/main/java/io/netty/handler/codec/http2/Http2LocalFlowController.java

```diff
@@ -39,22 +41,25 @@
      * @throws Http2Exception if any flow control errors are encountered.
      */
     void receiveFlowControlledFrame(ChannelHandlerContext ctx, Http2Stream stream, ByteBuf data, int padding,
-            boolean endOfStream) throws Http2Exception;
+                                    boolean endOfStream) throws Http2Exception;
 
     /**
-     * Indicates that the application has consumed a number of bytes for the given stream and is
-     * therefore ready to receive more data from the remote endpoint. The application must consume
-     * any bytes that it receives or the flow control window will collapse. Consuming bytes enables
-     * the flow controller to send {@code WINDOW_UPDATE} to restore a portion of the flow control
-     * window for the stream.
+     * Indicates that the application has consumed a number of bytes for the given stream and is therefore ready to
+     * receive more data from the remote endpoint. The application must consume any bytes that it receives or the flow
+     * control window will collapse. Consuming bytes enables the flow controller to send {@code WINDOW_UPDATE} to
+     * restore a portion of the flow control window for the stream.
+     * <p/>
+     * If {@code stream} is closed (i.e. {@link Http2Stream#state()} method returns {@link
+     * io.netty.handler.codec.http2.Http2Stream.State#CLOSED}), the consumed bytes are only restored to the connection
```

Comment:
do we need to fully qualify this? `io.netty.handler.codec.http2.Http2Stream.State` -> `Http2Stream.State`?